app/matplotlib.ipynb

```diff
@@ -233,24 +343,53 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
-   "outputs": [],
    "source": [
     "x = np.linspace(0, 5, 100)\n",
     "y1 = np.sin(x)\n",
     "y2 = np.cos(x)\n",
     "\n",
-    "# write your code here\n"
+    "# write your code here\n",
+    "fig = plt.figure()\n",
+    "\n",
+    "ax1 = fig.add_subplot(1, 2, 1)\n",
+    "ax1.plot(x, y1)\n",
+    "ax1.set_title(\"Sinus wave\")\n",
+    "ax1.set_xlabel(\"X\")\n",
+    "ax1.set_ylabel(\"sin(x)\")\n",
+    "\n",
+    "ax2=fig.add_subplot(1, 2, 2)\n",
+    "ax2.plot(x, y2)\n",
```

Comment:
The cos plot does not specify a color. Consider using `ax2.plot(x, y2, color='red')` to set a specific color for the cos plot.